package.json

```diff
@@ -8,10 +8,14 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.4",
     "@expo/vector-icons": "^10.0.6",
     "@react-native-community/masked-view": "^0.1.7",
+    "apollo-boost": "^0.4.7",
+    "dotenv": "^8.2.0",
```

Comment:
we ended up not being able to use dotenv on React apps, so wen should probably remove this